<!DOCTYPE html>

<html lang="en">
<head><script type="text/javascript" src="https://web.archive.org/_static/js/bundle-playback.js?v=HxkREWBo" charset="utf-8"></script>
<script type="text/javascript" src="https://web.archive.org/_static/js/wombat.js?v=txqj7nKC" charset="utf-8"></script>
<script>window.RufflePlayer=window.RufflePlayer||{};window.RufflePlayer.config={"autoplay":"on","unmuteOverlay":"hidden"};</script>
<script type="text/javascript" src="https://web.archive.org/_static/js/ruffle/ruffle.js"></script>
<script type="text/javascript">
    __wm.init("https://web.archive.org/web");
  __wm.wombat("http://www.neuroelectro.org:80/neuron/18/","20151005110219","https://web.archive.org/","web","/_static/",
	      "1444042939");
</script>
<link rel="stylesheet" type="text/css" href="https://web.archive.org/_static/css/banner-styles.css?v=S1zqJCYt" />
<link rel="stylesheet" type="text/css" href="https://web.archive.org/_static/css/iconochive.css?v=3PDvdIFv" />
<!-- End Wayback Rewrite JS Include -->

    <title>NeuroElectro :: Cerebellum Purkinje cell</title>
	
    <link rel="shortcut icon" href="https://web.archive.org/web/20151005110219im_/http://www.neuroelectro.org/static/src/favicon.ico">
    <link rel="stylesheet" type="text/css" href="https://web.archive.org/web/20151005110219cs_/http://www.neuroelectro.org/static/bootstrap_old/css/bootstrap.css">
    <link rel="stylesheet" type="text/css" href="https://web.archive.org/web/20151005110219cs_/http://www.neuroelectro.org/static/src/local.css"/>
    
    
	<script type="text/javascript" charset="utf-8" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/DataTables/media/js/jquery.js"></script>
 		<script type="text/javascript" charset="utf-8" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/DataTables/media/js/jquery.dataTables.js"></script>
 		<style type="text/css" title="currentStyle">
 		    @import "https://web.archive.org/web/20151005110219cs_/http://www.neuroelectro.org/static/src/DataTables/media/css/demo_table.css";
 		</style>
 		<!--<script language="javascript" type="text/javascript" src="/static/src/jquery.min.js"></script>-->
 		<script language="javascript" type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/jquery.jqplot.min.js"></script>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/plugins/jqplot.canvasTextRenderer.min.js"></script>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/plugins/jqplot.canvasAxisTickRenderer.min.js"></script>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/plugins/jqplot.categoryAxisRenderer.min.js"></script>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/plugins/jqplot.highlighter.js"></script>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/plugins/jqplot.canvasAxisLabelRenderer.min.js"></script>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/plugins/jqplot.cursor.min.js"></script>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/plugins/jqplot.pointLabels.min.js"></script>
 		<!--<script type="text/javascript" src="/static/src/plugins/jqplot.eventListenerHooks.min.js"></script>-->
 		<link rel="stylesheet" type="text/css" href="https://web.archive.org/web/20151005110219cs_/http://www.neuroelectro.org/static/src/jquery.jqplot.min.css"/>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/plugins/jqplot.ohlcRenderer.min.js"></script>
 		    <!-- Add jQuery library -->
 		<!--<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.8/jquery.min.js"></script>-->
 		
 		<!-- Add mousewheel plugin (this is optional) -->
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/fancybox/lib/jquery.mousewheel-3.0.6.pack.js"></script>
 		
 		<!-- Add fancyBox -->
 		<link rel="stylesheet" href="https://web.archive.org/web/20151005110219cs_/http://www.neuroelectro.org/static/src/fancybox/source/jquery.fancybox.css?v=2.1.3" type="text/css" media="screen"/>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/fancybox/source/jquery.fancybox.pack.js?v=2.1.3"></script>
 		
 		<!-- Optionally add helpers - button, thumbnail and/or media -->
 		<link rel="stylesheet" href="https://web.archive.org/web/20151005110219cs_/http://www.neuroelectro.org/static/src/fancybox/source/helpers/jquery.fancybox-buttons.css?v=1.0.5" type="text/css" media="screen"/>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/fancybox/source/helpers/jquery.fancybox-buttons.js?v=1.0.5"></script>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/fancybox/source/helpers/jquery.fancybox-media.js?v=1.0.5"></script>
 		
 		<link rel="stylesheet" href="https://web.archive.org/web/20151005110219cs_/http://www.neuroelectro.org/static/src/fancybox/source/helpers/jquery.fancybox-thumbs.css?v=1.0.7" type="text/css" media="screen"/>
 		<script type="text/javascript" src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/fancybox/source/helpers/jquery.fancybox-thumbs.js?v=1.0.7"></script>
 		<script src="https://web.archive.org/web/20151005110219js_/http://code.jquery.com/ui/1.10.4/jquery-ui.js"></script>
 		<link href="https://web.archive.org/web/20151005110219cs_/http://www.neuroelectro.org/static//src/NinjaRadial/NinjaRadial.css" rel="stylesheet" type="text/css"/>
		<script src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static//src/NinjaRadial/NinjaRadial.js" type="text/javascript"></script>
	
	
	
<style type="text/css">
.jqplot-axis {
    font-family: Arial, Helvetica, sans-serif;
    font-size: 90%; 
}
.jqplot-highlighter-tooltip {
    background-color: #B4EEB4;
    filter:alpha(opacity=90);
    opacity:0.9;
    padding: 10px;
    border-radius: 5px;
    font-family: Arial, Helvetica, sans-serif;
    z-index: 2;
}
.ui-tooltip {
    background-color: #B4EEB4;
    font-family: Arial, Helvetica, sans-serif;
    width:350px;
    padding:10px;
    border-radius: 5px;
    opacity:0.9;
    z-index: 2;
}
#ephys_prop_table {
    margin-left: 30px;
    margin-right: 30px;
}
.submit_button{
    margin-left: 50px;
    margin-right: 50px;
}
</style>

     
 	<style>
 		#maintanence_message{
 			color: red;
 		    text-align: center;
 		}
 	</style>
	
    
    <!--stuff for google tracking-->
    <script type="text/javascript">

      var _gaq = _gaq || [];
      _gaq.push(['_setAccount', 'UA-35182339-1']);
      _gaq.push(['_trackPageview']);

      (function() {
        var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
        ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
        var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
      })();
	
    </script>
</head>



<script>




$(document).ready(function(){
  var xticks = ["Reports", "Mean+SD", "All neurons"]
    
    //var minVal = parseFloat([[2, 124.465, 54.849, &#39;10&#39;, &#39;2&#39;]]) - parseFloat([[2, 69.6153416145552], [2, 179.3146583854448]]);
    //var maxVal = parseFloat([[2, 124.465, 54.849, &#39;10&#39;, &#39;2&#39;]]) + parseFloat([[2, 69.6153416145552], [2, 179.3146583854448]]);
  //bDat = [ [ [1.5, minVal], [3.5, minVal]],
  //[ [1.5, maxVal], [3.5, maxVal]] ]; 

  var yaxisLabel = "input resistance (Mâ„¦)";  

  var highlighterPos = 'ne';
    
    

  //var dataSeries = [[[1.0172804505564537, 23.3, 'Cerebellum Purkinje cell', 'Purkinje-cell-restricted restoration of Kv3.3 function restores</br>complex spikes and rescues motor coordination in Kcnc3 mutants.', 'Hurlock EC; McMahon A; Joho RH', 'J. Neurosci.', '2008', '2069'], [0.9879602777578405, 175.0, 'Cerebellum Purkinje cell', 'Impaired synaptic plasticity and cAMP response element-binding protein</br>activation in Ca2+/calmodulin-dependent protein kinase type IV/Gr-</br>deficient mice.', 'Ho N; Liauw JA; Blaeser F; Wei F; Hanissian S; Muglia LM; Wozniak DF;</br>Nardi A; Arvin KL; Holtzman DM; Linden DJ; Zhuo M; Muglia LJ; Chatila</br>TA', 'J. Neurosci.', '2000', '3119'], [0.9994757667908966, 181.0, 'Cerebellum Purkinje cell', 'Activation of presynaptic cAMP-dependent protein kinase is required</br>for induction of cerebellar long-term potentiation.', 'Linden DJ; Ahn S', 'J. Neurosci.', '1999', '3226'], [0.9909720511090427, 101.9, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376'], [0.9900220114710336, 140.0, 'Cerebellum Purkinje cell', 'Aminopyridines correct early dysfunction and delay neurodegeneration</br>in a mouse model of spinocerebellar ataxia type 1.', "Hourez R; Servais L; Orduz D; Gall D; Millard I; de Kerchove d'Exaerde</br>A; Cheron G; Orr HT; Pandolfo M; Schiffmann SN", 'J. Neurosci.', '2011', '1575'], [1.0109080818702487, 95.4, 'Cerebellum Purkinje cell', 'In vivo transduction of murine cerebellar Purkinje cells by HIV-</br>derived lentiviral vectors.', 'Torashima T; Okoyama S; Nishizaki T; Hirai H', 'Brain Res.', '2006', '21092'], [1.0056236393695483, 64.3, 'Cerebellum Purkinje cell', 'Selective and regulated gene expression in murine Purkinje cells by in</br>utero electroporation.', 'Nishiyama J; Hayashi Y; Nomura T; Miura E; Kakegawa W; Yuzaki M', 'Eur. J. Neurosci.', '2012', '16674'], [1.0089903951953616, 197.0, 'Cerebellum Purkinje cell', 'Chronic ethanol treatment alters AMPA-induced calcium signals in</br>developing Purkinje neurons.', 'Netzeband JG; Trotter C; Parsons KL; Gruol DL', 'Brain Res.', '1999', '22081'], [1.0093550593252263, 91.75, 'Cerebellum Purkinje cell', 'Membrane excitability and fear conditioning in cerebellar Purkinje</br>cell.', 'Zhu L; Scelfo B; Tempia F; Sacchetti B; Strata P', 'Neuroscience', '2006', '25316'], [0.9905196205488724, 175.0, 'Cerebellum Purkinje cell', 'Impaired cerebellar long-term potentiation in type I adenylyl cyclase</br>mutant mice.', 'Storm DR; Hansel C; Hacker B; Parent A; Linden DJ', 'Neuron', '1998', '21965']], [[2, 69.6153416145552], [2, 179.3146583854448]], [[2, 124.465, 54.849, '10', '2']], [[3, 20.233093652408], [3, 627.844203916036]], [[3, 324.038, 303.805, '91', '2']]];
  var dataSeries = [[[]], [[1.0172804505564537, 23.3, 'Cerebellum Purkinje cell', 'Purkinje-cell-restricted restoration of Kv3.3 function restores</br>complex spikes and rescues motor coordination in Kcnc3 mutants.', 'Hurlock EC; McMahon A; Joho RH', 'J. Neurosci.', '2008', '2069'], [0.9879602777578405, 175.0, 'Cerebellum Purkinje cell', 'Impaired synaptic plasticity and cAMP response element-binding protein</br>activation in Ca2+/calmodulin-dependent protein kinase type IV/Gr-</br>deficient mice.', 'Ho N; Liauw JA; Blaeser F; Wei F; Hanissian S; Muglia LM; Wozniak DF;</br>Nardi A; Arvin KL; Holtzman DM; Linden DJ; Zhuo M; Muglia LJ; Chatila</br>TA', 'J. Neurosci.', '2000', '3119'], [0.9994757667908966, 181.0, 'Cerebellum Purkinje cell', 'Activation of presynaptic cAMP-dependent protein kinase is required</br>for induction of cerebellar long-term potentiation.', 'Linden DJ; Ahn S', 'J. Neurosci.', '1999', '3226'], [0.9909720511090427, 101.9, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376'], [0.9900220114710336, 140.0, 'Cerebellum Purkinje cell', 'Aminopyridines correct early dysfunction and delay neurodegeneration</br>in a mouse model of spinocerebellar ataxia type 1.', "Hourez R; Servais L; Orduz D; Gall D; Millard I; de Kerchove d'Exaerde</br>A; Cheron G; Orr HT; Pandolfo M; Schiffmann SN", 'J. Neurosci.', '2011', '1575'], [1.0109080818702487, 95.4, 'Cerebellum Purkinje cell', 'In vivo transduction of murine cerebellar Purkinje cells by HIV-</br>derived lentiviral vectors.', 'Torashima T; Okoyama S; Nishizaki T; Hirai H', 'Brain Res.', '2006', '21092'], [1.0056236393695483, 64.3, 'Cerebellum Purkinje cell', 'Selective and regulated gene expression in murine Purkinje cells by in</br>utero electroporation.', 'Nishiyama J; Hayashi Y; Nomura T; Miura E; Kakegawa W; Yuzaki M', 'Eur. J. Neurosci.', '2012', '16674'], [1.0089903951953616, 197.0, 'Cerebellum Purkinje cell', 'Chronic ethanol treatment alters AMPA-induced calcium signals in</br>developing Purkinje neurons.', 'Netzeband JG; Trotter C; Parsons KL; Gruol DL', 'Brain Res.', '1999', '22081'], [1.0093550593252263, 91.75, 'Cerebellum Purkinje cell', 'Membrane excitability and fear conditioning in cerebellar Purkinje</br>cell.', 'Zhu L; Scelfo B; Tempia F; Sacchetti B; Strata P', 'Neuroscience', '2006', '25316'], [0.9905196205488724, 175.0, 'Cerebellum Purkinje cell', 'Impaired cerebellar long-term potentiation in type I adenylyl cyclase</br>mutant mice.', 'Storm DR; Hansel C; Hacker B; Parent A; Linden DJ', 'Neuron', '1998', '21965']], [[2, 69.6153416145552], [2, 179.3146583854448]], [[2, 124.465, 54.849, '10', '2']], [[3, 20.233093652408], [3, 627.844203916036]], [[3, 324.038, 303.805, '91', '2']]];
  var plot1b = $.jqplot('chart0', dataSeries, {
    //title: "input resistance",
    series:[
        {
            showLine:false,
            color:"#EAA228",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                yvalues: 6,
                tooltipFadeSpeed: .1,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (Mâ„¦)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:false,
            color:"#4bb2c5",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 6,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (Mâ„¦)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:true,
            color: "#c5b47f",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#c5b47f",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (Mâ„¦)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (Mâ„¦)</td></tr> \
                <tr><td># articles: </td><td>%s</td></tr></table>'
            },
        },
        {
            showLine:true,
            color: "#FA8072",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#FA8072",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (Mâ„¦)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (Mâ„¦)</td></tr> \
                <tr><td># neurons: </td><td>%s</td></tr></table>'
            },
        },
    ],
    axesDefaults: {
        tickRenderer: $.jqplot.CanvasAxisTickRenderer,
        tickOptions: {
          fontSize: '12pt',
        }
    },
    axes: {
      xaxis: {
        renderer: $.jqplot.CategoryAxisRenderer,
        labelOptions:{
            fontFamily: 'arial',
        },
        min: .5,
        max: 3.5,
        ticks: xticks,
        tickOptions: {
          angle: -90,
        }
      },
      yaxis: {
        label: yaxisLabel,
        labelRenderer: $.jqplot.CanvasAxisLabelRenderer,
        autoscale : true,
        labelOptions:{
            fontFamily: 'arial',
            fontSize: '12pt',
        },
      },
    },
    highlighter: {
        show: true,
    },
    cursor:{
        show: true, 
        zoom: true,
        looseZoom: true,
          showTooltip:false, 
          followMouse: true, 
          showTooltipOutsideZoom: false, 
          constrainOutsideZoom: true,
    }, 
  });
        $('#chart0').bind('jqplotDataClick', 
            function (ev, seriesIndex, pointIndex, data) {
                if (seriesIndex == 0 || seriesIndex == 1)
                    {
                        var data_pt_ind = data[7];
                        var linkStr;
                        if (data_pt_ind < 0) // this is a hack to redirect non-data table ext data to the article page
                        {
                            linkStr = "/article/" + (-data_pt_ind);
                        }
                        else 
                        {
                            linkStr = "/data_table/" + data_pt_ind;
                        }
                        window.open(linkStr, '_self');
                    }
                else
                    {
                        var data_pt_ind = data[4];
                        var linkStr = "/ephys_prop/" + data_pt_ind;
                        window.open(linkStr, '_self');                    
                    }
                
                
            }
        );
        //assign each jqplot's y-axis label an id the same as the id of the ephys prop
        $('#chart0').find(".jqplot-yaxis canvas")[0].id = 2;
        $('#chart0').find(".jqplot-yaxis canvas")[0].title = 'Common Definition: Input resistance measured at steady-state voltage response to current injection <br/> <br/> Standardization Criteria: Values corrected for differences in units, but are otherwise unchanged. Refer to individual articles for specific definitions and calculation methodologies.';
        

      
    //var minVal = parseFloat([[2, -60.487, 6.608, &#39;4&#39;, &#39;3&#39;]]) - parseFloat([[2, -67.09643098390352], [2, -53.87856901609648]]);
    //var maxVal = parseFloat([[2, -60.487, 6.608, &#39;4&#39;, &#39;3&#39;]]) + parseFloat([[2, -67.09643098390352], [2, -53.87856901609648]]);
  //bDat = [ [ [1.5, minVal], [3.5, minVal]],
  //[ [1.5, maxVal], [3.5, maxVal]] ]; 

  var yaxisLabel = "resting membrane potential (mV)";  

  var highlighterPos = 'ne';
    
    

  //var dataSeries = [[[1.001407156022161, -58.55, 'Cerebellum Purkinje cell', 'Aminopyridines correct early dysfunction and delay neurodegeneration</br>in a mouse model of spinocerebellar ataxia type 1.', "Hourez R; Servais L; Orduz D; Gall D; Millard I; de Kerchove d'Exaerde</br>A; Cheron G; Orr HT; Pandolfo M; Schiffmann SN", 'J. Neurosci.', '2011', '1575'], [1.0012202282565978, -63.4, 'Cerebellum Purkinje cell', 'In vivo transduction of murine cerebellar Purkinje cells by HIV-</br>derived lentiviral vectors.', 'Torashima T; Okoyama S; Nishizaki T; Hirai H', 'Brain Res.', '2006', '21092'], [1.0136388475986489, -51.0, 'Cerebellum Purkinje cell', 'Chronic ethanol treatment alters AMPA-induced calcium signals in</br>developing Purkinje neurons.', 'Netzeband JG; Trotter C; Parsons KL; Gruol DL', 'Brain Res.', '1999', '22081'], [1.0061765614175064, -69.0, 'Cerebellum Purkinje cell', 'Impaired cerebellar long-term potentiation in type I adenylyl cyclase</br>mutant mice.', 'Storm DR; Hansel C; Hacker B; Parent A; Linden DJ', 'Neuron', '1998', '21965']], [[2, -67.09643098390352], [2, -53.87856901609648]], [[2, -60.487, 6.608, '4', '3']], [[3, -71.0344752253084], [3, -54.6180911878442]], [[3, -62.826, 8.208, '87', '3']]];
  var dataSeries = [[[]], [[1.001407156022161, -58.55, 'Cerebellum Purkinje cell', 'Aminopyridines correct early dysfunction and delay neurodegeneration</br>in a mouse model of spinocerebellar ataxia type 1.', "Hourez R; Servais L; Orduz D; Gall D; Millard I; de Kerchove d'Exaerde</br>A; Cheron G; Orr HT; Pandolfo M; Schiffmann SN", 'J. Neurosci.', '2011', '1575'], [1.0012202282565978, -63.4, 'Cerebellum Purkinje cell', 'In vivo transduction of murine cerebellar Purkinje cells by HIV-</br>derived lentiviral vectors.', 'Torashima T; Okoyama S; Nishizaki T; Hirai H', 'Brain Res.', '2006', '21092'], [1.0136388475986489, -51.0, 'Cerebellum Purkinje cell', 'Chronic ethanol treatment alters AMPA-induced calcium signals in</br>developing Purkinje neurons.', 'Netzeband JG; Trotter C; Parsons KL; Gruol DL', 'Brain Res.', '1999', '22081'], [1.0061765614175064, -69.0, 'Cerebellum Purkinje cell', 'Impaired cerebellar long-term potentiation in type I adenylyl cyclase</br>mutant mice.', 'Storm DR; Hansel C; Hacker B; Parent A; Linden DJ', 'Neuron', '1998', '21965']], [[2, -67.09643098390352], [2, -53.87856901609648]], [[2, -60.487, 6.608, '4', '3']], [[3, -71.0344752253084], [3, -54.6180911878442]], [[3, -62.826, 8.208, '87', '3']]];
  var plot1b = $.jqplot('chart1', dataSeries, {
    //title: "resting membrane potential",
    series:[
        {
            showLine:false,
            color:"#EAA228",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                yvalues: 6,
                tooltipFadeSpeed: .1,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (mV)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:false,
            color:"#4bb2c5",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 6,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (mV)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:true,
            color: "#c5b47f",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#c5b47f",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (mV)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (mV)</td></tr> \
                <tr><td># articles: </td><td>%s</td></tr></table>'
            },
        },
        {
            showLine:true,
            color: "#FA8072",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#FA8072",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (mV)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (mV)</td></tr> \
                <tr><td># neurons: </td><td>%s</td></tr></table>'
            },
        },
    ],
    axesDefaults: {
        tickRenderer: $.jqplot.CanvasAxisTickRenderer,
        tickOptions: {
          fontSize: '12pt',
        }
    },
    axes: {
      xaxis: {
        renderer: $.jqplot.CategoryAxisRenderer,
        labelOptions:{
            fontFamily: 'arial',
        },
        min: .5,
        max: 3.5,
        ticks: xticks,
        tickOptions: {
          angle: -90,
        }
      },
      yaxis: {
        label: yaxisLabel,
        labelRenderer: $.jqplot.CanvasAxisLabelRenderer,
        autoscale : true,
        labelOptions:{
            fontFamily: 'arial',
            fontSize: '12pt',
        },
      },
    },
    highlighter: {
        show: true,
    },
    cursor:{
        show: true, 
        zoom: true,
        looseZoom: true,
          showTooltip:false, 
          followMouse: true, 
          showTooltipOutsideZoom: false, 
          constrainOutsideZoom: true,
    }, 
  });
        $('#chart1').bind('jqplotDataClick', 
            function (ev, seriesIndex, pointIndex, data) {
                if (seriesIndex == 0 || seriesIndex == 1)
                    {
                        var data_pt_ind = data[7];
                        var linkStr;
                        if (data_pt_ind < 0) // this is a hack to redirect non-data table ext data to the article page
                        {
                            linkStr = "/article/" + (-data_pt_ind);
                        }
                        else 
                        {
                            linkStr = "/data_table/" + data_pt_ind;
                        }
                        window.open(linkStr, '_self');
                    }
                else
                    {
                        var data_pt_ind = data[4];
                        var linkStr = "/ephys_prop/" + data_pt_ind;
                        window.open(linkStr, '_self');                    
                    }
                
                
            }
        );
        //assign each jqplot's y-axis label an id the same as the id of the ephys prop
        $('#chart1').find(".jqplot-yaxis canvas")[0].id = 3;
        $('#chart1').find(".jqplot-yaxis canvas")[0].title = 'Common Definition: Membrane potential at the onset of whole-cell recording <br/> <br/> Standardization Criteria: Values unchanged from those reported. Refer to individual articles for specific definitions and calculation methodologies.';
        

      
    //var minVal = parseFloat([[2, -51.92, 4.005, &#39;4&#39;, &#39;7&#39;]]) - parseFloat([[2, -55.92552118955823], [2, -47.91447881044177]]);
    //var maxVal = parseFloat([[2, -51.92, 4.005, &#39;4&#39;, &#39;7&#39;]]) + parseFloat([[2, -55.92552118955823], [2, -47.91447881044177]]);
  //bDat = [ [ [1.5, minVal], [3.5, minVal]],
  //[ [1.5, maxVal], [3.5, maxVal]] ]; 

  var yaxisLabel = "spike threshold (mV)";  

  var highlighterPos = 'ne';
    
    

  //var dataSeries = [[[0.9988378332191501, -54.9, 'Cerebellum Purkinje cell', 'Submillisecond optical reporting of membrane potential in situ using a</br>neuronal tracer dye.', 'Bradley J; Luo R; Otis TS; DiGregorio DA', 'J. Neurosci.', '2009', '1949'], [1.0015001619508344, -53.0, 'Cerebellum Purkinje cell', 'Robustness of burst firing in dissociated purkinje neurons with acute</br>or long-term reductions in sodium conductance.', 'Swensen AM; Bean BP', 'J. Neurosci.', '2005', '2484'], [1.0049529766516552, -45.1, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376'], [0.989607707907831, -54.68, 'Cerebellum Purkinje cell', 'Membrane excitability and fear conditioning in cerebellar Purkinje</br>cell.', 'Zhu L; Scelfo B; Tempia F; Sacchetti B; Strata P', 'Neuroscience', '2006', '25316']], [[2, -55.92552118955823], [2, -47.91447881044177]], [[2, -51.92, 4.005, '4', '7']], [[3, -48.4232311684134], [3, -35.2269455260086]], [[3, -41.825, 6.598, '73', '7']]];
  var dataSeries = [[[]], [[0.9988378332191501, -54.9, 'Cerebellum Purkinje cell', 'Submillisecond optical reporting of membrane potential in situ using a</br>neuronal tracer dye.', 'Bradley J; Luo R; Otis TS; DiGregorio DA', 'J. Neurosci.', '2009', '1949'], [1.0015001619508344, -53.0, 'Cerebellum Purkinje cell', 'Robustness of burst firing in dissociated purkinje neurons with acute</br>or long-term reductions in sodium conductance.', 'Swensen AM; Bean BP', 'J. Neurosci.', '2005', '2484'], [1.0049529766516552, -45.1, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376'], [0.989607707907831, -54.68, 'Cerebellum Purkinje cell', 'Membrane excitability and fear conditioning in cerebellar Purkinje</br>cell.', 'Zhu L; Scelfo B; Tempia F; Sacchetti B; Strata P', 'Neuroscience', '2006', '25316']], [[2, -55.92552118955823], [2, -47.91447881044177]], [[2, -51.92, 4.005, '4', '7']], [[3, -48.4232311684134], [3, -35.2269455260086]], [[3, -41.825, 6.598, '73', '7']]];
  var plot1b = $.jqplot('chart2', dataSeries, {
    //title: "spike threshold",
    series:[
        {
            showLine:false,
            color:"#EAA228",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                yvalues: 6,
                tooltipFadeSpeed: .1,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (mV)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:false,
            color:"#4bb2c5",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 6,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (mV)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:true,
            color: "#c5b47f",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#c5b47f",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (mV)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (mV)</td></tr> \
                <tr><td># articles: </td><td>%s</td></tr></table>'
            },
        },
        {
            showLine:true,
            color: "#FA8072",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#FA8072",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (mV)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (mV)</td></tr> \
                <tr><td># neurons: </td><td>%s</td></tr></table>'
            },
        },
    ],
    axesDefaults: {
        tickRenderer: $.jqplot.CanvasAxisTickRenderer,
        tickOptions: {
          fontSize: '12pt',
        }
    },
    axes: {
      xaxis: {
        renderer: $.jqplot.CategoryAxisRenderer,
        labelOptions:{
            fontFamily: 'arial',
        },
        min: .5,
        max: 3.5,
        ticks: xticks,
        tickOptions: {
          angle: -90,
        }
      },
      yaxis: {
        label: yaxisLabel,
        labelRenderer: $.jqplot.CanvasAxisLabelRenderer,
        autoscale : true,
        labelOptions:{
            fontFamily: 'arial',
            fontSize: '12pt',
        },
      },
    },
    highlighter: {
        show: true,
    },
    cursor:{
        show: true, 
        zoom: true,
        looseZoom: true,
          showTooltip:false, 
          followMouse: true, 
          showTooltipOutsideZoom: false, 
          constrainOutsideZoom: true,
    }, 
  });
        $('#chart2').bind('jqplotDataClick', 
            function (ev, seriesIndex, pointIndex, data) {
                if (seriesIndex == 0 || seriesIndex == 1)
                    {
                        var data_pt_ind = data[7];
                        var linkStr;
                        if (data_pt_ind < 0) // this is a hack to redirect non-data table ext data to the article page
                        {
                            linkStr = "/article/" + (-data_pt_ind);
                        }
                        else 
                        {
                            linkStr = "/data_table/" + data_pt_ind;
                        }
                        window.open(linkStr, '_self');
                    }
                else
                    {
                        var data_pt_ind = data[4];
                        var linkStr = "/ephys_prop/" + data_pt_ind;
                        window.open(linkStr, '_self');                    
                    }
                
                
            }
        );
        //assign each jqplot's y-axis label an id the same as the id of the ephys prop
        $('#chart2').find(".jqplot-yaxis canvas")[0].id = 7;
        $('#chart2').find(".jqplot-yaxis canvas")[0].title = 'Common Definition: Voltage at which AP is initiated (as assessed by measuring rising slope of membrane voltage) <br/> <br/> Standardization Criteria: Values are unchanged from those reported. Refer to individual articles for specific calculation methodology, such as voltage slope criterion for AP threshold determination or which APs are used to define spike theshold (i.e., evoked using rheobase current, via synaptic input, etc.).';
        

      
    //var minVal = parseFloat([[2, 77.903, 0.99, &#39;3&#39;, &#39;5&#39;]]) - parseFloat([[2, 76.91269941732014], [2, 78.89396724934645]]);
    //var maxVal = parseFloat([[2, 77.903, 0.99, &#39;3&#39;, &#39;5&#39;]]) + parseFloat([[2, 76.91269941732014], [2, 78.89396724934645]]);
  //bDat = [ [ [1.5, minVal], [3.5, minVal]],
  //[ [1.5, maxVal], [3.5, maxVal]] ]; 

  var yaxisLabel = "spike amplitude (mV)";  

  var highlighterPos = 'ne';
    
    
        highlighterPos = 'nw';
    

  //var dataSeries = [[[1.013881999325258, 79.3, 'Cerebellum Purkinje cell', 'Submillisecond optical reporting of membrane potential in situ using a</br>neuronal tracer dye.', 'Bradley J; Luo R; Otis TS; DiGregorio DA', 'J. Neurosci.', '2009', '1949'], [1.0036943935274156, 77.3, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376'], [1.0000096896680317, 77.11, 'Cerebellum Purkinje cell', 'Membrane excitability and fear conditioning in cerebellar Purkinje</br>cell.', 'Zhu L; Scelfo B; Tempia F; Sacchetti B; Strata P', 'Neuroscience', '2006', '25316']], [[2, 76.91269941732014], [2, 78.89396724934645]], [[2, 77.903, 0.99, '3', '5']], [[3, 55.2818812983306], [3, 83.069060239787]], [[3, 69.175, 13.893, '72', '5']]];
  var dataSeries = [[[]], [[1.013881999325258, 79.3, 'Cerebellum Purkinje cell', 'Submillisecond optical reporting of membrane potential in situ using a</br>neuronal tracer dye.', 'Bradley J; Luo R; Otis TS; DiGregorio DA', 'J. Neurosci.', '2009', '1949'], [1.0036943935274156, 77.3, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376'], [1.0000096896680317, 77.11, 'Cerebellum Purkinje cell', 'Membrane excitability and fear conditioning in cerebellar Purkinje</br>cell.', 'Zhu L; Scelfo B; Tempia F; Sacchetti B; Strata P', 'Neuroscience', '2006', '25316']], [[2, 76.91269941732014], [2, 78.89396724934645]], [[2, 77.903, 0.99, '3', '5']], [[3, 55.2818812983306], [3, 83.069060239787]], [[3, 69.175, 13.893, '72', '5']]];
  var plot1b = $.jqplot('chart3', dataSeries, {
    //title: "spike amplitude",
    series:[
        {
            showLine:false,
            color:"#EAA228",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                yvalues: 6,
                tooltipFadeSpeed: .1,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (mV)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:false,
            color:"#4bb2c5",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 6,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (mV)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:true,
            color: "#c5b47f",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#c5b47f",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (mV)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (mV)</td></tr> \
                <tr><td># articles: </td><td>%s</td></tr></table>'
            },
        },
        {
            showLine:true,
            color: "#FA8072",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#FA8072",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (mV)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (mV)</td></tr> \
                <tr><td># neurons: </td><td>%s</td></tr></table>'
            },
        },
    ],
    axesDefaults: {
        tickRenderer: $.jqplot.CanvasAxisTickRenderer,
        tickOptions: {
          fontSize: '12pt',
        }
    },
    axes: {
      xaxis: {
        renderer: $.jqplot.CategoryAxisRenderer,
        labelOptions:{
            fontFamily: 'arial',
        },
        min: .5,
        max: 3.5,
        ticks: xticks,
        tickOptions: {
          angle: -90,
        }
      },
      yaxis: {
        label: yaxisLabel,
        labelRenderer: $.jqplot.CanvasAxisLabelRenderer,
        autoscale : true,
        labelOptions:{
            fontFamily: 'arial',
            fontSize: '12pt',
        },
      },
    },
    highlighter: {
        show: true,
    },
    cursor:{
        show: true, 
        zoom: true,
        looseZoom: true,
          showTooltip:false, 
          followMouse: true, 
          showTooltipOutsideZoom: false, 
          constrainOutsideZoom: true,
    }, 
  });
        $('#chart3').bind('jqplotDataClick', 
            function (ev, seriesIndex, pointIndex, data) {
                if (seriesIndex == 0 || seriesIndex == 1)
                    {
                        var data_pt_ind = data[7];
                        var linkStr;
                        if (data_pt_ind < 0) // this is a hack to redirect non-data table ext data to the article page
                        {
                            linkStr = "/article/" + (-data_pt_ind);
                        }
                        else 
                        {
                            linkStr = "/data_table/" + data_pt_ind;
                        }
                        window.open(linkStr, '_self');
                    }
                else
                    {
                        var data_pt_ind = data[4];
                        var linkStr = "/ephys_prop/" + data_pt_ind;
                        window.open(linkStr, '_self');                    
                    }
                
                
            }
        );
        //assign each jqplot's y-axis label an id the same as the id of the ephys prop
        $('#chart3').find(".jqplot-yaxis canvas")[0].id = 5;
        $('#chart3').find(".jqplot-yaxis canvas")[0].title = 'Common Definition: Voltage indicating height of action potential. Usually calculated as the difference between AP peak and AP threshold voltages. Commonly measured using first AP in train at rheobase current. <br/> <br/> Standardization Criteria: Values manually standardized case-by-case to correspond to calculation definition of difference between AP peak and AP threshold. Since authors use different APs for defining spike amplitude (e.g., evoked using rheobase current, via synaptic input, etc.), please refer to individual articles for specific methodology.';
        

      
    //var minVal = parseFloat([[2, 0.332, 0.112, &#39;5&#39;, &#39;6&#39;]]) - parseFloat([[2, 0.219526311855873], [2, 0.444873688144127]]);
    //var maxVal = parseFloat([[2, 0.332, 0.112, &#39;5&#39;, &#39;6&#39;]]) + parseFloat([[2, 0.219526311855873], [2, 0.444873688144127]]);
  //bDat = [ [ [1.5, minVal], [3.5, minVal]],
  //[ [1.5, maxVal], [3.5, maxVal]] ]; 

  var yaxisLabel = "spike half-width (ms)";  

  var highlighterPos = 'ne';
    
          highlighterPos = 'nw';
    
    

  //var dataSeries = [[[0.9972510322072985, 0.431, 'Cerebellum Purkinje cell', 'Submillisecond optical reporting of membrane potential in situ using a</br>neuronal tracer dye.', 'Bradley J; Luo R; Otis TS; DiGregorio DA', 'J. Neurosci.', '2009', '1949'], [1.0125035013565535, 0.15, 'Cerebellum Purkinje cell', 'Purkinje-cell-restricted restoration of Kv3.3 function restores</br>complex spikes and rescues motor coordination in Kcnc3 mutants.', 'Hurlock EC; McMahon A; Joho RH', 'J. Neurosci.', '2008', '2069'], [0.9996222270273175, 0.41, 'Cerebellum Purkinje cell', 'Robustness of burst firing in dissociated purkinje neurons with acute</br>or long-term reductions in sodium conductance.', 'Swensen AM; Bean BP', 'J. Neurosci.', '2005', '2484'], [1.0136337587710678, 0.27, 'Cerebellum Purkinje cell', 'Aminopyridines correct early dysfunction and delay neurodegeneration</br>in a mouse model of spinocerebellar ataxia type 1.', "Hourez R; Servais L; Orduz D; Gall D; Millard I; de Kerchove d'Exaerde</br>A; Cheron G; Orr HT; Pandolfo M; Schiffmann SN", 'J. Neurosci.', '2011', '1575'], [1.0075676927801445, 0.23, 'Cerebellum Purkinje cell', 'Aminopyridines correct early dysfunction and delay neurodegeneration</br>in a mouse model of spinocerebellar ataxia type 1.', "Hourez R; Servais L; Orduz D; Gall D; Millard I; de Kerchove d'Exaerde</br>A; Cheron G; Orr HT; Pandolfo M; Schiffmann SN", 'J. Neurosci.', '2011', '1575'], [0.9896727615277786, 0.42, 'Cerebellum Purkinje cell', 'The leaner P/Q-type calcium channel mutation renders cerebellar</br>Purkinje neurons hyper-excitable and eliminates Ca2+-Na+ spike bursts.', 'Ovsepian SV; Friel DD', 'Eur. J. Neurosci.', '2008', '17398']], [[2, 0.219526311855873], [2, 0.444873688144127]], [[2, 0.332, 0.112, '5', '6']], [[3, 0.6604266947514089], [3, 1.5683165195222308]], [[3, 1.114, 0.453, '69', '6']]];
  var dataSeries = [[[]], [[0.9972510322072985, 0.431, 'Cerebellum Purkinje cell', 'Submillisecond optical reporting of membrane potential in situ using a</br>neuronal tracer dye.', 'Bradley J; Luo R; Otis TS; DiGregorio DA', 'J. Neurosci.', '2009', '1949'], [1.0125035013565535, 0.15, 'Cerebellum Purkinje cell', 'Purkinje-cell-restricted restoration of Kv3.3 function restores</br>complex spikes and rescues motor coordination in Kcnc3 mutants.', 'Hurlock EC; McMahon A; Joho RH', 'J. Neurosci.', '2008', '2069'], [0.9996222270273175, 0.41, 'Cerebellum Purkinje cell', 'Robustness of burst firing in dissociated purkinje neurons with acute</br>or long-term reductions in sodium conductance.', 'Swensen AM; Bean BP', 'J. Neurosci.', '2005', '2484'], [1.0136337587710678, 0.27, 'Cerebellum Purkinje cell', 'Aminopyridines correct early dysfunction and delay neurodegeneration</br>in a mouse model of spinocerebellar ataxia type 1.', "Hourez R; Servais L; Orduz D; Gall D; Millard I; de Kerchove d'Exaerde</br>A; Cheron G; Orr HT; Pandolfo M; Schiffmann SN", 'J. Neurosci.', '2011', '1575'], [1.0075676927801445, 0.23, 'Cerebellum Purkinje cell', 'Aminopyridines correct early dysfunction and delay neurodegeneration</br>in a mouse model of spinocerebellar ataxia type 1.', "Hourez R; Servais L; Orduz D; Gall D; Millard I; de Kerchove d'Exaerde</br>A; Cheron G; Orr HT; Pandolfo M; Schiffmann SN", 'J. Neurosci.', '2011', '1575'], [0.9896727615277786, 0.42, 'Cerebellum Purkinje cell', 'The leaner P/Q-type calcium channel mutation renders cerebellar</br>Purkinje neurons hyper-excitable and eliminates Ca2+-Na+ spike bursts.', 'Ovsepian SV; Friel DD', 'Eur. J. Neurosci.', '2008', '17398']], [[2, 0.219526311855873], [2, 0.444873688144127]], [[2, 0.332, 0.112, '5', '6']], [[3, 0.6604266947514089], [3, 1.5683165195222308]], [[3, 1.114, 0.453, '69', '6']]];
  var plot1b = $.jqplot('chart4', dataSeries, {
    //title: "spike half-width",
    series:[
        {
            showLine:false,
            color:"#EAA228",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                yvalues: 6,
                tooltipFadeSpeed: .1,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (ms)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:false,
            color:"#4bb2c5",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 6,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (ms)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:true,
            color: "#c5b47f",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#c5b47f",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (ms)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (ms)</td></tr> \
                <tr><td># articles: </td><td>%s</td></tr></table>'
            },
        },
        {
            showLine:true,
            color: "#FA8072",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#FA8072",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (ms)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (ms)</td></tr> \
                <tr><td># neurons: </td><td>%s</td></tr></table>'
            },
        },
    ],
    axesDefaults: {
        tickRenderer: $.jqplot.CanvasAxisTickRenderer,
        tickOptions: {
          fontSize: '12pt',
        }
    },
    axes: {
      xaxis: {
        renderer: $.jqplot.CategoryAxisRenderer,
        labelOptions:{
            fontFamily: 'arial',
        },
        min: .5,
        max: 3.5,
        ticks: xticks,
        tickOptions: {
          angle: -90,
        }
      },
      yaxis: {
        label: yaxisLabel,
        labelRenderer: $.jqplot.CanvasAxisLabelRenderer,
        autoscale : true,
        labelOptions:{
            fontFamily: 'arial',
            fontSize: '12pt',
        },
      },
    },
    highlighter: {
        show: true,
    },
    cursor:{
        show: true, 
        zoom: true,
        looseZoom: true,
          showTooltip:false, 
          followMouse: true, 
          showTooltipOutsideZoom: false, 
          constrainOutsideZoom: true,
    }, 
  });
        $('#chart4').bind('jqplotDataClick', 
            function (ev, seriesIndex, pointIndex, data) {
                if (seriesIndex == 0 || seriesIndex == 1)
                    {
                        var data_pt_ind = data[7];
                        var linkStr;
                        if (data_pt_ind < 0) // this is a hack to redirect non-data table ext data to the article page
                        {
                            linkStr = "/article/" + (-data_pt_ind);
                        }
                        else 
                        {
                            linkStr = "/data_table/" + data_pt_ind;
                        }
                        window.open(linkStr, '_self');
                    }
                else
                    {
                        var data_pt_ind = data[4];
                        var linkStr = "/ephys_prop/" + data_pt_ind;
                        window.open(linkStr, '_self');                    
                    }
                
                
            }
        );
        //assign each jqplot's y-axis label an id the same as the id of the ephys prop
        $('#chart4').find(".jqplot-yaxis canvas")[0].id = 6;
        $('#chart4').find(".jqplot-yaxis canvas")[0].title = 'Common Definition: Calculated as the AP duration at the membrane voltage halfway between AP threshold and AP peak. Most commonly calculated using first AP in train at rheobase. current. <br/> <br/> Standardization Criteria: Values manually standardized case-by-case to correspond to calculation definition of AP width at half-max amplitude. Since authors use different APs for defining spike half-width (i.e., evoked using rheobase current, via synaptic input, etc.), please refer to individual articles for specific methodology.';
        

      
    //var minVal = parseFloat([[2, 14.69, 4.88, &#39;4&#39;, &#39;16&#39;]]) - parseFloat([[2, 9.809805331751608], [2, 19.57019466824839]]);
    //var maxVal = parseFloat([[2, 14.69, 4.88, &#39;4&#39;, &#39;16&#39;]]) + parseFloat([[2, 9.809805331751608], [2, 19.57019466824839]]);
  //bDat = [ [ [1.5, minVal], [3.5, minVal]],
  //[ [1.5, maxVal], [3.5, maxVal]] ]; 

  var yaxisLabel = "AHP amplitude (mV)";  

  var highlighterPos = 'ne';
    
    

  //var dataSeries = [[[0.9977073468815164, 21.0, 'Cerebellum Purkinje cell', 'Robustness of burst firing in dissociated purkinje neurons with acute</br>or long-term reductions in sodium conductance.', 'Swensen AM; Bean BP', 'J. Neurosci.', '2005', '2484'], [1.0027054880639221, 12.2, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376'], [1.0071671791280024, 8.2, 'Cerebellum Purkinje cell', 'The leaner P/Q-type calcium channel mutation renders cerebellar</br>Purkinje neurons hyper-excitable and eliminates Ca2+-Na+ spike bursts.', 'Ovsepian SV; Friel DD', 'Eur. J. Neurosci.', '2008', '17398'], [1.0124007317702572, 17.36, 'Cerebellum Purkinje cell', 'Membrane excitability and fear conditioning in cerebellar Purkinje</br>cell.', 'Zhu L; Scelfo B; Tempia F; Sacchetti B; Strata P', 'Neuroscience', '2006', '25316']], [[2, 9.809805331751608], [2, 19.57019466824839]], [[2, 14.69, 4.88, '4', '16']], [[3, 8.404650046069719], [3, 20.90412110341568]], [[3, 14.654, 6.249, '50', '16']]];
  var dataSeries = [[[]], [[0.9977073468815164, 21.0, 'Cerebellum Purkinje cell', 'Robustness of burst firing in dissociated purkinje neurons with acute</br>or long-term reductions in sodium conductance.', 'Swensen AM; Bean BP', 'J. Neurosci.', '2005', '2484'], [1.0027054880639221, 12.2, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376'], [1.0071671791280024, 8.2, 'Cerebellum Purkinje cell', 'The leaner P/Q-type calcium channel mutation renders cerebellar</br>Purkinje neurons hyper-excitable and eliminates Ca2+-Na+ spike bursts.', 'Ovsepian SV; Friel DD', 'Eur. J. Neurosci.', '2008', '17398'], [1.0124007317702572, 17.36, 'Cerebellum Purkinje cell', 'Membrane excitability and fear conditioning in cerebellar Purkinje</br>cell.', 'Zhu L; Scelfo B; Tempia F; Sacchetti B; Strata P', 'Neuroscience', '2006', '25316']], [[2, 9.809805331751608], [2, 19.57019466824839]], [[2, 14.69, 4.88, '4', '16']], [[3, 8.404650046069719], [3, 20.90412110341568]], [[3, 14.654, 6.249, '50', '16']]];
  var plot1b = $.jqplot('chart5', dataSeries, {
    //title: "AHP amplitude",
    series:[
        {
            showLine:false,
            color:"#EAA228",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                yvalues: 6,
                tooltipFadeSpeed: .1,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (mV)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:false,
            color:"#4bb2c5",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 6,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (mV)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:true,
            color: "#c5b47f",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#c5b47f",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (mV)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (mV)</td></tr> \
                <tr><td># articles: </td><td>%s</td></tr></table>'
            },
        },
        {
            showLine:true,
            color: "#FA8072",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#FA8072",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (mV)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (mV)</td></tr> \
                <tr><td># neurons: </td><td>%s</td></tr></table>'
            },
        },
    ],
    axesDefaults: {
        tickRenderer: $.jqplot.CanvasAxisTickRenderer,
        tickOptions: {
          fontSize: '12pt',
        }
    },
    axes: {
      xaxis: {
        renderer: $.jqplot.CategoryAxisRenderer,
        labelOptions:{
            fontFamily: 'arial',
        },
        min: .5,
        max: 3.5,
        ticks: xticks,
        tickOptions: {
          angle: -90,
        }
      },
      yaxis: {
        label: yaxisLabel,
        labelRenderer: $.jqplot.CanvasAxisLabelRenderer,
        autoscale : true,
        labelOptions:{
            fontFamily: 'arial',
            fontSize: '12pt',
        },
      },
    },
    highlighter: {
        show: true,
    },
    cursor:{
        show: true, 
        zoom: true,
        looseZoom: true,
          showTooltip:false, 
          followMouse: true, 
          showTooltipOutsideZoom: false, 
          constrainOutsideZoom: true,
    }, 
  });
        $('#chart5').bind('jqplotDataClick', 
            function (ev, seriesIndex, pointIndex, data) {
                if (seriesIndex == 0 || seriesIndex == 1)
                    {
                        var data_pt_ind = data[7];
                        var linkStr;
                        if (data_pt_ind < 0) // this is a hack to redirect non-data table ext data to the article page
                        {
                            linkStr = "/article/" + (-data_pt_ind);
                        }
                        else 
                        {
                            linkStr = "/data_table/" + data_pt_ind;
                        }
                        window.open(linkStr, '_self');
                    }
                else
                    {
                        var data_pt_ind = data[4];
                        var linkStr = "/ephys_prop/" + data_pt_ind;
                        window.open(linkStr, '_self');                    
                    }
                
                
            }
        );
        //assign each jqplot's y-axis label an id the same as the id of the ephys prop
        $('#chart5').find(".jqplot-yaxis canvas")[0].id = 16;
        $('#chart5').find(".jqplot-yaxis canvas")[0].title = 'Common Definition: Calculated as the voltage difference between AP threshold and AP trough. Commonly defined using first AP in train at rheobase current. <br/> <br/> Standardization Criteria: Values have been algorithmically standardized to correspond to calculation definition of difference between AP threshold and AHP trough. Since authors use different APs for defining AHP amplitudes (i.e., evoked using rheobase current, via synaptic input, etc.), please refer to individual articles for specific methodology.';
        

      
    //var minVal = parseFloat([[2, 706.3, 593.7, &#39;2&#39;, &#39;8&#39;]]) - parseFloat([[2, 112.59999999999991], [2, 1300.0]]);
    //var maxVal = parseFloat([[2, 706.3, 593.7, &#39;2&#39;, &#39;8&#39;]]) + parseFloat([[2, 112.59999999999991], [2, 1300.0]]);
  //bDat = [ [ [1.5, minVal], [3.5, minVal]],
  //[ [1.5, maxVal], [3.5, maxVal]] ]; 

  var yaxisLabel = "rheobase (pA)";  

  var highlighterPos = 'ne';
    
    

  //var dataSeries = [[[1.0060744372168822, 1300.0, 'Cerebellum Purkinje cell', 'Submillisecond optical reporting of membrane potential in situ using a</br>neuronal tracer dye.', 'Bradley J; Luo R; Otis TS; DiGregorio DA', 'J. Neurosci.', '2009', '1949'], [0.9954323419959996, 112.6, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376']], [[2, 112.59999999999991], [2, 1300.0]], [[2, 706.3, 593.7, '2', '8']], [[3, -605.3574204833811], [3, 1606.038698261159]], [[3, 500.34, 1105.698, '30', '8']]];
  var dataSeries = [[[]], [[1.0060744372168822, 1300.0, 'Cerebellum Purkinje cell', 'Submillisecond optical reporting of membrane potential in situ using a</br>neuronal tracer dye.', 'Bradley J; Luo R; Otis TS; DiGregorio DA', 'J. Neurosci.', '2009', '1949'], [0.9954323419959996, 112.6, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376']], [[2, 112.59999999999991], [2, 1300.0]], [[2, 706.3, 593.7, '2', '8']], [[3, -605.3574204833811], [3, 1606.038698261159]], [[3, 500.34, 1105.698, '30', '8']]];
  var plot1b = $.jqplot('chart6', dataSeries, {
    //title: "rheobase",
    series:[
        {
            showLine:false,
            color:"#EAA228",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                yvalues: 6,
                tooltipFadeSpeed: .1,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (pA)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:false,
            color:"#4bb2c5",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 6,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (pA)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:true,
            color: "#c5b47f",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#c5b47f",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (pA)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (pA)</td></tr> \
                <tr><td># articles: </td><td>%s</td></tr></table>'
            },
        },
        {
            showLine:true,
            color: "#FA8072",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#FA8072",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (pA)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (pA)</td></tr> \
                <tr><td># neurons: </td><td>%s</td></tr></table>'
            },
        },
    ],
    axesDefaults: {
        tickRenderer: $.jqplot.CanvasAxisTickRenderer,
        tickOptions: {
          fontSize: '12pt',
        }
    },
    axes: {
      xaxis: {
        renderer: $.jqplot.CategoryAxisRenderer,
        labelOptions:{
            fontFamily: 'arial',
        },
        min: .5,
        max: 3.5,
        ticks: xticks,
        tickOptions: {
          angle: -90,
        }
      },
      yaxis: {
        label: yaxisLabel,
        labelRenderer: $.jqplot.CanvasAxisLabelRenderer,
        autoscale : true,
        labelOptions:{
            fontFamily: 'arial',
            fontSize: '12pt',
        },
      },
    },
    highlighter: {
        show: true,
    },
    cursor:{
        show: true, 
        zoom: true,
        looseZoom: true,
          showTooltip:false, 
          followMouse: true, 
          showTooltipOutsideZoom: false, 
          constrainOutsideZoom: true,
    }, 
  });
        $('#chart6').bind('jqplotDataClick', 
            function (ev, seriesIndex, pointIndex, data) {
                if (seriesIndex == 0 || seriesIndex == 1)
                    {
                        var data_pt_ind = data[7];
                        var linkStr;
                        if (data_pt_ind < 0) // this is a hack to redirect non-data table ext data to the article page
                        {
                            linkStr = "/article/" + (-data_pt_ind);
                        }
                        else 
                        {
                            linkStr = "/data_table/" + data_pt_ind;
                        }
                        window.open(linkStr, '_self');
                    }
                else
                    {
                        var data_pt_ind = data[4];
                        var linkStr = "/ephys_prop/" + data_pt_ind;
                        window.open(linkStr, '_self');                    
                    }
                
                
            }
        );
        //assign each jqplot's y-axis label an id the same as the id of the ephys prop
        $('#chart6').find(".jqplot-yaxis canvas")[0].id = 8;
        $('#chart6').find(".jqplot-yaxis canvas")[0].title = 'Common Definition: Minimum current injected somatically required to fire AP <br/> <br/> Standardization Criteria: Values corrected for differences in reported units, but are otherwise unchanged. Refer to individual articles for specific definitions and calculation methodologies.';
        

      
    //var minVal = parseFloat([[2, 596.01, 263.322, &#39;4&#39;, &#39;1&#39;]]) - parseFloat([[2, 332.687579192352], [2, 859.332420807648]]);
    //var maxVal = parseFloat([[2, 596.01, 263.322, &#39;4&#39;, &#39;1&#39;]]) + parseFloat([[2, 332.687579192352], [2, 859.332420807648]]);
  //bDat = [ [ [1.5, minVal], [3.5, minVal]],
  //[ [1.5, maxVal], [3.5, maxVal]] ]; 

  var yaxisLabel = "cell capacitance (pF)";  

  var highlighterPos = 'ne';
    
    

  //var dataSeries = [[[0.9939943345201688, 543.7, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376'], [0.9972702182041766, 334.14, 'Cerebellum Purkinje cell', 'Aminopyridines correct early dysfunction and delay neurodegeneration</br>in a mouse model of spinocerebellar ataxia type 1.', "Hourez R; Servais L; Orduz D; Gall D; Millard I; de Kerchove d'Exaerde</br>A; Cheron G; Orr HT; Pandolfo M; Schiffmann SN", 'J. Neurosci.', '2011', '1575'], [0.9996394863912688, 473.2, 'Cerebellum Purkinje cell', 'In vivo transduction of murine cerebellar Purkinje cells by HIV-</br>derived lentiviral vectors.', 'Torashima T; Okoyama S; Nishizaki T; Hirai H', 'Brain Res.', '2006', '21092'], [0.9997877367064277, 1033.0, 'Cerebellum Purkinje cell', 'Selective and regulated gene expression in murine Purkinje cells by in</br>utero electroporation.', 'Nishiyama J; Hayashi Y; Nomura T; Miura E; Kakegawa W; Yuzaki M', 'Eur. J. Neurosci.', '2012', '16674']], [[2, 332.687579192352], [2, 859.332420807648]], [[2, 596.01, 263.322, '4', '1']], [[3, -277.40577269431196], [3, 604.52637098491]], [[3, 163.56, 440.966, '39', '1']]];
  var dataSeries = [[[]], [[0.9939943345201688, 543.7, 'Cerebellum Purkinje cell', 'Lobule-specific membrane excitability of cerebellar Purkinje cells.', 'Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ', 'J. Physiol. (Lond.)', '2012', '8376'], [0.9972702182041766, 334.14, 'Cerebellum Purkinje cell', 'Aminopyridines correct early dysfunction and delay neurodegeneration</br>in a mouse model of spinocerebellar ataxia type 1.', "Hourez R; Servais L; Orduz D; Gall D; Millard I; de Kerchove d'Exaerde</br>A; Cheron G; Orr HT; Pandolfo M; Schiffmann SN", 'J. Neurosci.', '2011', '1575'], [0.9996394863912688, 473.2, 'Cerebellum Purkinje cell', 'In vivo transduction of murine cerebellar Purkinje cells by HIV-</br>derived lentiviral vectors.', 'Torashima T; Okoyama S; Nishizaki T; Hirai H', 'Brain Res.', '2006', '21092'], [0.9997877367064277, 1033.0, 'Cerebellum Purkinje cell', 'Selective and regulated gene expression in murine Purkinje cells by in</br>utero electroporation.', 'Nishiyama J; Hayashi Y; Nomura T; Miura E; Kakegawa W; Yuzaki M', 'Eur. J. Neurosci.', '2012', '16674']], [[2, 332.687579192352], [2, 859.332420807648]], [[2, 596.01, 263.322, '4', '1']], [[3, -277.40577269431196], [3, 604.52637098491]], [[3, 163.56, 440.966, '39', '1']]];
  var plot1b = $.jqplot('chart7', dataSeries, {
    //title: "cell capacitance",
    series:[
        {
            showLine:false,
            color:"#EAA228",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                yvalues: 6,
                tooltipFadeSpeed: .1,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (pF)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:false,
            color:"#4bb2c5",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 6,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (pF)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:true,
            color: "#c5b47f",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#c5b47f",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (pF)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (pF)</td></tr> \
                <tr><td># articles: </td><td>%s</td></tr></table>'
            },
        },
        {
            showLine:true,
            color: "#FA8072",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#FA8072",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (pF)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (pF)</td></tr> \
                <tr><td># neurons: </td><td>%s</td></tr></table>'
            },
        },
    ],
    axesDefaults: {
        tickRenderer: $.jqplot.CanvasAxisTickRenderer,
        tickOptions: {
          fontSize: '12pt',
        }
    },
    axes: {
      xaxis: {
        renderer: $.jqplot.CategoryAxisRenderer,
        labelOptions:{
            fontFamily: 'arial',
        },
        min: .5,
        max: 3.5,
        ticks: xticks,
        tickOptions: {
          angle: -90,
        }
      },
      yaxis: {
        label: yaxisLabel,
        labelRenderer: $.jqplot.CanvasAxisLabelRenderer,
        autoscale : true,
        labelOptions:{
            fontFamily: 'arial',
            fontSize: '12pt',
        },
      },
    },
    highlighter: {
        show: true,
    },
    cursor:{
        show: true, 
        zoom: true,
        looseZoom: true,
          showTooltip:false, 
          followMouse: true, 
          showTooltipOutsideZoom: false, 
          constrainOutsideZoom: true,
    }, 
  });
        $('#chart7').bind('jqplotDataClick', 
            function (ev, seriesIndex, pointIndex, data) {
                if (seriesIndex == 0 || seriesIndex == 1)
                    {
                        var data_pt_ind = data[7];
                        var linkStr;
                        if (data_pt_ind < 0) // this is a hack to redirect non-data table ext data to the article page
                        {
                            linkStr = "/article/" + (-data_pt_ind);
                        }
                        else 
                        {
                            linkStr = "/data_table/" + data_pt_ind;
                        }
                        window.open(linkStr, '_self');
                    }
                else
                    {
                        var data_pt_ind = data[4];
                        var linkStr = "/ephys_prop/" + data_pt_ind;
                        window.open(linkStr, '_self');                    
                    }
                
                
            }
        );
        //assign each jqplot's y-axis label an id the same as the id of the ephys prop
        $('#chart7').find(".jqplot-yaxis canvas")[0].id = 1;
        $('#chart7').find(".jqplot-yaxis canvas")[0].title = 'Common Definition: Neuron capacitance, typically measured by dividing membrane time constant by membrane resistance <br/> <br/> Standardization Criteria: Values corrected for differences in units, but are otherwise unchanged. Refer to individual articles for specific definitions and calculation methodologies.';
        

      
    //var minVal = parseFloat([[2, 146.5, 0.0, &#39;1&#39;, &#39;17&#39;]]) - parseFloat([[2, 146.5], [2, 146.5]]);
    //var maxVal = parseFloat([[2, 146.5, 0.0, &#39;1&#39;, &#39;17&#39;]]) + parseFloat([[2, 146.5], [2, 146.5]]);
  //bDat = [ [ [1.5, minVal], [3.5, minVal]],
  //[ [1.5, maxVal], [3.5, maxVal]] ]; 

  var yaxisLabel = "FI slope (Hz/nA)";  

  var highlighterPos = 'ne';
    
    
        highlighterPos = 'nw';
    

  //var dataSeries = [[[1.0023124843165898, 146.5, 'Cerebellum Purkinje cell', 'Purkinje-cell-restricted restoration of Kv3.3 function restores</br>complex spikes and rescues motor coordination in Kcnc3 mutants.', 'Hurlock EC; McMahon A; Joho RH', 'J. Neurosci.', '2008', '2069']], [[2, 146.5], [2, 146.5]], [[2, 146.5, 0.0, '1', '17']], [[3, 6.018848882324008], [3, 215.80915111767598]], [[3, 110.914, 104.895, '19', '17']]];
  var dataSeries = [[[]], [[1.0023124843165898, 146.5, 'Cerebellum Purkinje cell', 'Purkinje-cell-restricted restoration of Kv3.3 function restores</br>complex spikes and rescues motor coordination in Kcnc3 mutants.', 'Hurlock EC; McMahon A; Joho RH', 'J. Neurosci.', '2008', '2069']], [[2, 146.5], [2, 146.5]], [[2, 146.5, 0.0, '1', '17']], [[3, 6.018848882324008], [3, 215.80915111767598]], [[3, 110.914, 104.895, '19', '17']]];
  var plot1b = $.jqplot('chart8', dataSeries, {
    //title: "FI slope",
    series:[
        {
            showLine:false,
            color:"#EAA228",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                yvalues: 6,
                tooltipFadeSpeed: .1,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (Hz/nA)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:false,
            color:"#4bb2c5",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 6,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (Hz/nA)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:true,
            color: "#c5b47f",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#c5b47f",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (Hz/nA)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (Hz/nA)</td></tr> \
                <tr><td># articles: </td><td>%s</td></tr></table>'
            },
        },
        {
            showLine:true,
            color: "#FA8072",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#FA8072",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (Hz/nA)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (Hz/nA)</td></tr> \
                <tr><td># neurons: </td><td>%s</td></tr></table>'
            },
        },
    ],
    axesDefaults: {
        tickRenderer: $.jqplot.CanvasAxisTickRenderer,
        tickOptions: {
          fontSize: '12pt',
        }
    },
    axes: {
      xaxis: {
        renderer: $.jqplot.CategoryAxisRenderer,
        labelOptions:{
            fontFamily: 'arial',
        },
        min: .5,
        max: 3.5,
        ticks: xticks,
        tickOptions: {
          angle: -90,
        }
      },
      yaxis: {
        label: yaxisLabel,
        labelRenderer: $.jqplot.CanvasAxisLabelRenderer,
        autoscale : true,
        labelOptions:{
            fontFamily: 'arial',
            fontSize: '12pt',
        },
      },
    },
    highlighter: {
        show: true,
    },
    cursor:{
        show: true, 
        zoom: true,
        looseZoom: true,
          showTooltip:false, 
          followMouse: true, 
          showTooltipOutsideZoom: false, 
          constrainOutsideZoom: true,
    }, 
  });
        $('#chart8').bind('jqplotDataClick', 
            function (ev, seriesIndex, pointIndex, data) {
                if (seriesIndex == 0 || seriesIndex == 1)
                    {
                        var data_pt_ind = data[7];
                        var linkStr;
                        if (data_pt_ind < 0) // this is a hack to redirect non-data table ext data to the article page
                        {
                            linkStr = "/article/" + (-data_pt_ind);
                        }
                        else 
                        {
                            linkStr = "/data_table/" + data_pt_ind;
                        }
                        window.open(linkStr, '_self');
                    }
                else
                    {
                        var data_pt_ind = data[4];
                        var linkStr = "/ephys_prop/" + data_pt_ind;
                        window.open(linkStr, '_self');                    
                    }
                
                
            }
        );
        //assign each jqplot's y-axis label an id the same as the id of the ephys prop
        $('#chart8').find(".jqplot-yaxis canvas")[0].id = 17;
        $('#chart8').find(".jqplot-yaxis canvas")[0].title = 'Common Definition: Slope of the frequency-current relationship <br/> <br/> Standardization Criteria: Values corrected to reflect differences in units, but are otherwise unchanged. Refer to individual articles for definition and calculation methodology.';
        

      
    //var minVal = parseFloat([[2, 53.6, 12.6, &#39;2&#39;, &#39;26&#39;]]) - parseFloat([[2, 41.0], [2, 66.2]]);
    //var maxVal = parseFloat([[2, 53.6, 12.6, &#39;2&#39;, &#39;26&#39;]]) + parseFloat([[2, 41.0], [2, 66.2]]);
  //bDat = [ [ [1.5, minVal], [3.5, minVal]],
  //[ [1.5, maxVal], [3.5, maxVal]] ]; 

  var yaxisLabel = "spike peak (mV)";  

  var highlighterPos = 'ne';
    
          highlighterPos = 'nw';
    
    

  //var dataSeries = [[[1.0013925313828413, 41.0, 'Cerebellum Purkinje cell', 'Robustness of burst firing in dissociated purkinje neurons with acute</br>or long-term reductions in sodium conductance.', 'Swensen AM; Bean BP', 'J. Neurosci.', '2005', '2484'], [1.0089632144658296, 66.2, 'Cerebellum Purkinje cell', 'The leaner P/Q-type calcium channel mutation renders cerebellar</br>Purkinje neurons hyper-excitable and eliminates Ca2+-Na+ spike bursts.', 'Ovsepian SV; Friel DD', 'Eur. J. Neurosci.', '2008', '17398']], [[2, 41.0], [2, 66.2]], [[2, 53.6, 12.6, '2', '26']], [[3, 19.9305335979006], [3, 42.804883068766]], [[3, 31.367, 11.437, '8', '26']]];
  var dataSeries = [[[]], [[1.0013925313828413, 41.0, 'Cerebellum Purkinje cell', 'Robustness of burst firing in dissociated purkinje neurons with acute</br>or long-term reductions in sodium conductance.', 'Swensen AM; Bean BP', 'J. Neurosci.', '2005', '2484'], [1.0089632144658296, 66.2, 'Cerebellum Purkinje cell', 'The leaner P/Q-type calcium channel mutation renders cerebellar</br>Purkinje neurons hyper-excitable and eliminates Ca2+-Na+ spike bursts.', 'Ovsepian SV; Friel DD', 'Eur. J. Neurosci.', '2008', '17398']], [[2, 41.0], [2, 66.2]], [[2, 53.6, 12.6, '2', '26']], [[3, 19.9305335979006], [3, 42.804883068766]], [[3, 31.367, 11.437, '8', '26']]];
  var plot1b = $.jqplot('chart9', dataSeries, {
    //title: "spike peak",
    series:[
        {
            showLine:false,
            color:"#EAA228",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                yvalues: 6,
                tooltipFadeSpeed: .1,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (mV)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:false,
            color:"#4bb2c5",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 6,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Value: </td><td>%s (mV)</td></tr> \
                <tr><td>Neuron:</td><td>%s</td></tr> \
                <tr><td>Title:</td><td>%s</td></tr> \
                <tr><td>Authors:</td><td>%s</td></tr> \
                <tr><td>Journal:</td><td>%s, %s</td></tr> \
                </table>'
            },
        },
        {
            showLine:true,
            color: "#c5b47f",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#c5b47f",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (mV)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (mV)</td></tr> \
                <tr><td># articles: </td><td>%s</td></tr></table>'
            },
        },
        {
            showLine:true,
            color: "#FA8072",
            showMarker: false,
            showTooltip: false,
        },
        {
            showLine:false,
            color: "#FA8072",
            highlighter: {
                sizeAdjust: 12,
                tooltipAxes: 'y',
                tooltipLocation: highlighterPos,
                useAxesFormatters: false,
                tooltipFadeSpeed: .1,
                yvalues: 3,
                formatString: '<table class="jqplot-highlighter"> \
                <tr><td>Mean value: </td><td>%.2f (mV)</td></tr> \
                <tr><td>S.D. value: </td><td>%.2f (mV)</td></tr> \
                <tr><td># neurons: </td><td>%s</td></tr></table>'
            },
        },
    ],
    axesDefaults: {
        tickRenderer: $.jqplot.CanvasAxisTickRenderer,
        tickOptions: {
          fontSize: '12pt',
        }
    },
    axes: {
      xaxis: {
        renderer: $.jqplot.CategoryAxisRenderer,
        labelOptions:{
            fontFamily: 'arial',
        },
        min: .5,
        max: 3.5,
        ticks: xticks,
        tickOptions: {
          angle: -90,
        }
      },
      yaxis: {
        label: yaxisLabel,
        labelRenderer: $.jqplot.CanvasAxisLabelRenderer,
        autoscale : true,
        labelOptions:{
            fontFamily: 'arial',
            fontSize: '12pt',
        },
      },
    },
    highlighter: {
        show: true,
    },
    cursor:{
        show: true, 
        zoom: true,
        looseZoom: true,
          showTooltip:false, 
          followMouse: true, 
          showTooltipOutsideZoom: false, 
          constrainOutsideZoom: true,
    }, 
  });
        $('#chart9').bind('jqplotDataClick', 
            function (ev, seriesIndex, pointIndex, data) {
                if (seriesIndex == 0 || seriesIndex == 1)
                    {
                        var data_pt_ind = data[7];
                        var linkStr;
                        if (data_pt_ind < 0) // this is a hack to redirect non-data table ext data to the article page
                        {
                            linkStr = "/article/" + (-data_pt_ind);
                        }
                        else 
                        {
                            linkStr = "/data_table/" + data_pt_ind;
                        }
                        window.open(linkStr, '_self');
                    }
                else
                    {
                        var data_pt_ind = data[4];
                        var linkStr = "/ephys_prop/" + data_pt_ind;
                        window.open(linkStr, '_self');                    
                    }
                
                
            }
        );
        //assign each jqplot's y-axis label an id the same as the id of the ephys prop
        $('#chart9').find(".jqplot-yaxis canvas")[0].id = 26;
        $('#chart9').find(".jqplot-yaxis canvas")[0].title = 'Common Definition: Maximum voltage reached during AP <br/> <br/> Standardization Criteria: Values corrected to reflect differences in units, but are otherwise unchanged. Since authors use different APs for defining spike peak (e.g., evoked using rheobase current, via synaptic input, etc.), please refer to individual articles for specific methodology.';
        

    
/*

*/
    var oldTop = '0px';

    $(".jqplot-yaxis-label")
    .css({
        cursor: "pointer",
        zIndex: "1",
    })
    .click(function(){ 
        var ephys_label_id = $(this).attr('id');
        var linkStr = "/ephys_prop/" + ephys_label_id;
        window.open(linkStr, '_self');
    })
    .tooltip({
        track: true,
        position: { my: "left+15 center", at: "right center" },
        content: function() {
            return $(this).attr('title');
        },
    });


    
    $(".fancybox").fancybox();

    
    
    //Stuff for data table display
    $('#article_list_table').dataTable({
        "iDisplayLength": 5
    });
    var aTable = $('#article_list_table').dataTable();
    aTable.fnSort( [ [4,'desc']] );  

/*    $('#neuron_list_table').dataTable({
        "iDisplayLength": 5
    });
    var aTable = $('#neuron_list_table').dataTable();
    aTable.fnSort( [ [4,'desc']] );  */

        // function to keep x-axes active on zoom
/*   $(".chart").bind('jqplotZoom', function(ev, gridpos, datapos, plot, cursor){
        resetYLabels(gridpos)
    });*/
    // function to keep x-axes active after zoom out event
   $(".chart").bind('jqplotResetZoom', function(ev, gridpos, datapos, plot, cursor){
        resetYLabels(gridpos)
    });

   function resetYLabels(gridpos){
    var oldTop = '0px';
    var chart_id = gridpos.targetId;
    console.log(gridpos);
    $(chart_id + " .jqplot-yaxis-label")
    .css({
        cursor: "pointer",
        zIndex: "1"
    })
    .click(function(){ 
        var ephys_label_id = $(this).attr('id');
        var linkStr = "/ephys_prop/" + ephys_label_id;
        window.open(linkStr, '_self');
    })
    .tooltip({
        track: true,
        position: { my: "left+15 center", at: "right center" },
        content: function() {
            return $(this).attr('title');
        },
    });
   }

});


</script>



<div id="content">
<body>
	
    <table align="center" width="1200px" class="logo_bar">
        <tr>
            <td class="logo_bar">
                <div align="left">
					
					<a href="http://www.neuroelectro.org/">
				    	<img src="https://web.archive.org/web/20151005110219im_/http://www.neuroelectro.org/static/src/neuroelectro_logo_3.png" id="logo">
				    </a>
				    
                </div>
            </td>
            <td class="logo_bar">
                <div align="right">
                <a href="../publications.html">
                NeuroElectro Publications</a> &nbsp;
                <br>
                
                </div>
                
            </td>
        </tr>
    </table>

    

<html>

<head>

<!--<link rel="stylesheet" type="text/css" href="/static/src/local.css" />-->
<link href="https://web.archive.org/web/20151005110219cs_/http://www.neuroelectro.org/static/src/SimpleNavBarStyles.css" rel="stylesheet" type="text/css">
<script src="https://web.archive.org/web/20151005110219js_/http://www.neuroelectro.org/static/src/SimpleNavBarScripts.js" language="JavaScript" type="text/javascript"></script>

</head>

<body>




<!-- ############### #### Navigation Bar #### ############### -->


<!-- main nav bar titles -->
<!-- Note how the the closing angle bracket of each </a> tag runs up against the next <a> tag,
      to avoid leaving a gap between each menu title and the next one. -->
      
<div class="mynavbar">

<a class="navbartitle" id="t1" href="http://www.neuroelectro.org/" onmouseout="HideItem('about_submenu');" onmouseover="ShowItem('about_submenu');">About</a><a class="navbartitle" id="t2" href="../neuron/index.html" onmouseout="HideItem('neuron_submenu');" onmouseover="ShowItem('neuron_submenu');">Neuron Types</a><a class="navbartitle" id="t3" href="../ephys_prop/index.html">Electrophysiology Properties</a><a class="navbartitle" id="t4" href="../article/index.html">Articles</a><a class="navbartitle" id="t5" href="../faqs.html" onmouseout="HideItem('faqs_submenu');" onmouseover="ShowItem('faqs_submenu');">FAQs</a><a class="navbartitle" id="t6" href="../api/docs.html" onmouseout="HideItem('api_submenu');" onmouseover="ShowItem('api_submenu');">Data/API</a><a class="navbartitle" id="t7" href="../contribute.html">Contribute</a>

    
    <!-- about sub-menu, shown as needed  -->
<div class="submenu" id="about_submenu" onmouseover="ShowItem('about_submenu');" onmouseout="HideItem('about_submenu');">
  <div class="submenubox">
    <ul>
      <li><a href="../publications.html" class="submenlink">Publications</a></li>
      <!-- <li><a href="/citation_info" class="submenlink">How to Cite</a></li> -->
      <li><a href="https://groups.google.com/forum/#!forum/neuroelectro" class="submenlink" target="_blank">Mailing List</a></li>
      <li><a href="http://www.neuroelectro.org/contact_info" class="submenlink">Contact</a></li>
    </ul>
  </div>
</div>

    <!-- neuron sub-menu, shown as needed  -->
<div class="submenu" id="neuron_submenu" onmouseover="ShowItem('neuron_submenu');" onmouseout="HideItem('neuron_submenu');">
  <div class="submenubox">
    <ul>
      <li><a href="../neuron/index.html" class="submenlink">Neuron Listing</a></li>
      <li><a href="" class="submenlink">Neuron Clustering</a></li>
    </ul>
  </div>
</div>

<!-- api sub-menu, shown as needed  -->

<div class="submenu" id="api_submenu" onmouseover="ShowItem('api_submenu');" onmouseout="HideItem('api_submenu');">
  <div class="submenubox">
    <ul>
      <li><a href="../api/docs.html" class="submenlink">API Documentation</a></li>
      <!-- <li><a href='/static/src/neuroelectro_data_download.zip' class="submenlink" target="_blank">Data Download</a></li>  -->
      <li><a href="../publications.html" class="submenlink">How to Cite</a></li>
      <li><a href="https://groups.google.com/forum/#!forum/neuroelectro" class="submenlink" target="_blank">Ask a Question</a></li>
      <li><a href="https://github.com/neuroelectro" class="submenlink" target="_blank">Code Repository (GitHub)</a></li>
    </ul>
  </div>
</div>


</div>

<!-- ################## end of Navigation Bar ################### -->


</body>
</html>

    

    
    
<h1>
    Cerebellum Purkinje cell 
     
        <a class="fancybox" href="http://uri.neuinfo.org/nif/nifstd/sao471801888" data-fancybox-type="iframe">
                (Definition)
        </a>
    
</h1>

    <h2>Electrophysiological properties of <i>Cerebellum Purkinje cells</i> from literature:</h2>

    <table>
    <tr>
    <td>
    <ul>
    <lh>Legend:</lh>
    <li>Reports: Blue dots indicate human-curated values; Orange dots indicate non-human curated values</li>
    <li>Mean+SD: mean and standard deviation of human-curated neuron measurements</li>
    <li>All neurons: mean and standard deviation computed over all neurons in database</li>
    <lh>Interactivity:</lh>
    <li>Mouse over neuron report data points and click to view corresponding publication</li>
    <li>Mouse over y- axis labels to view definition or click to view values across neuron types</li>
    <li>Zoom in on a section of plot by dragging cursor. Zoom out by double clicking on plot.</li>
    <li>Legend: Blue dots = text-mined values human curated; Orange dots = text-mined values not human curated </li>
    </ul>  
    </td>
    <td>
        <div style="text-align:center">
        <a class="btn btn-large btn-info submit_button" href="../data/18.html/index.html"><i class="icon-th-list icon-white"></i> View data in table form</i></a>
        <br>
        <a class="fancybox btn btn-large btn-danger submit_button" data-fancybox-type="iframe" href="http://www.neuroelectro.org/nedm_comment_box/"><i class="icon-thumbs-down icon-white"></i> Report miscurated data</i></a>
        </div>
    </td>
    </tr>
    </table>
    </br>

    <table id="ephys_prop_table">
    
        
            <tr>
        
        <td>
        <br>
        <div>
        <div id="chart0" style="height:260px;width:220px;" class="chart"></div>
        <br>
        <div>
        </td>
        
    
        
        <td>
        <br>
        <div>
        <div id="chart1" style="height:260px;width:220px;" class="chart"></div>
        <br>
        <div>
        </td>
        
    
        
        <td>
        <br>
        <div>
        <div id="chart2" style="height:260px;width:220px;" class="chart"></div>
        <br>
        <div>
        </td>
        
    
        
        <td>
        <br>
        <div>
        <div id="chart3" style="height:260px;width:220px;" class="chart"></div>
        <br>
        <div>
        </td>
        
    
        
        <td>
        <br>
        <div>
        <div id="chart4" style="height:260px;width:220px;" class="chart"></div>
        <br>
        <div>
        </td>
        
            </tr>
            
        
    
        
            <tr>
        
        <td>
        <br>
        <div>
        <div id="chart5" style="height:260px;width:220px;" class="chart"></div>
        <br>
        <div>
        </td>
        
    
        
        <td>
        <br>
        <div>
        <div id="chart6" style="height:260px;width:220px;" class="chart"></div>
        <br>
        <div>
        </td>
        
    
        
        <td>
        <br>
        <div>
        <div id="chart7" style="height:260px;width:220px;" class="chart"></div>
        <br>
        <div>
        </td>
        
    
        
        <td>
        <br>
        <div>
        <div id="chart8" style="height:260px;width:220px;" class="chart"></div>
        <br>
        <div>
        </td>
        
    
        
        <td>
        <br>
        <div>
        <div id="chart9" style="height:260px;width:220px;" class="chart"></div>
        <br>
        <div>
        </td>
        
            </tr>
            
        
    
    </table>


<table>
    <tr>
        <td style="text-align: center;">
            <a class="btn btn-large btn-info submit_button" href="../data/18.html/index.html"><i class="icon-th-list icon-white"></i> View data in table form</i></a>
        </td>

        <td style="text-align: center;">
            
                    <a id="sign_in_button" class="btn btn-large btn-primary submit_button" href="http://www.neuroelectro.org/accounts/login?next=/neuron/18/"><i class="icon-user icon-white"></i> Contribute neuron data (login required)</a>
            
        </td>
    </tr>
</table>


<h1>Articles with extracted electrophysiology properties:</h1>
    <table id="article_list_table" class="display">
    <thead>
    <tr>
    <th>Article Title</th>
    <th>Authors</th>
    <th>Journal</th>
    <th>Year</th>
    <th>Electrophys values</th>
    <th>Neuron types</th>
    </tr>
    </thead>
    <tbody>
    
        <tr>
        <td>
        The leaner P/Q-type calcium channel mutation renders cerebellar Purkinje neurons hyper-excitable and eliminates Ca2+-Na+ spike bursts.
        (<a href="http://www.neuroelectro.org/article/5177/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/18093175/" target="_blank">PubMed</a>)
        </td>
        <td>Ovsepian SV; Friel DD</td>
        <td>Eur. J. Neurosci.</td>
        <td>2008</td>
        <td>3</a></td>
        <td>1</a></td>
        </tr>
    
        <tr>
        <td>
        Aminopyridines correct early dysfunction and delay neurodegeneration in a mouse model of spinocerebellar ataxia type 1.
        (<a href="http://www.neuroelectro.org/article/9102/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/21849540/" target="_blank">PubMed</a>)
        </td>
        <td>Hourez R; Servais L; Orduz D; Gall D; Millard I; de Kerchove d&#39;Exaerde A; Cheron G; Orr HT; Pandolfo M; Schiffmann SN</td>
        <td>J. Neurosci.</td>
        <td>2011</td>
        <td>9</a></td>
        <td>1</a></td>
        </tr>
    
        <tr>
        <td>
        Purkinje-cell-restricted restoration of Kv3.3 function restores complex spikes and rescues motor coordination in Kcnc3 mutants.
        (<a href="http://www.neuroelectro.org/article/9763/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/18448641/" target="_blank">PubMed</a>)
        </td>
        <td>Hurlock EC; McMahon A; Joho RH</td>
        <td>J. Neurosci.</td>
        <td>2008</td>
        <td>4</a></td>
        <td>1</a></td>
        </tr>
    
        <tr>
        <td>
        Robustness of burst firing in dissociated purkinje neurons with acute or long-term reductions in sodium conductance.
        (<a href="http://www.neuroelectro.org/article/9921/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/15814781/" target="_blank">PubMed</a>)
        </td>
        <td>Swensen AM; Bean BP</td>
        <td>J. Neurosci.</td>
        <td>2005</td>
        <td>4</a></td>
        <td>1</a></td>
        </tr>
    
        <tr>
        <td>
        Lobule-specific membrane excitability of cerebellar Purkinje cells.
        (<a href="http://www.neuroelectro.org/article/29594/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22083600/" target="_blank">PubMed</a>)
        </td>
        <td>Kim CH; Oh SH; Lee JH; Chang SO; Kim J; Kim SJ</td>
        <td>J. Physiol. (Lond.)</td>
        <td>2012</td>
        <td>6</a></td>
        <td>1</a></td>
        </tr>
    
        <tr>
        <td>
        Submillisecond optical reporting of membrane potential in situ using a neuronal tracer dye.
        (<a href="http://www.neuroelectro.org/article/35673/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/19625510/" target="_blank">PubMed</a>)
        </td>
        <td>Bradley J; Luo R; Otis TS; DiGregorio DA</td>
        <td>J. Neurosci.</td>
        <td>2009</td>
        <td>4</a></td>
        <td>1</a></td>
        </tr>
    
        <tr>
        <td>
        Impaired synaptic plasticity and cAMP response element-binding protein activation in Ca2+/calmodulin-dependent protein kinase type IV/Gr-deficient mice.
        (<a href="http://www.neuroelectro.org/article/37464/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/10964952/" target="_blank">PubMed</a>)
        </td>
        <td>Ho N; Liauw JA; Blaeser F; Wei F; Hanissian S; Muglia LM; Wozniak DF; Nardi A; Arvin KL; Holtzman DM; Linden DJ; Zhuo M; Muglia LJ; Chatila TA</td>
        <td>J. Neurosci.</td>
        <td>2000</td>
        <td>1</a></td>
        <td>1</a></td>
        </tr>
    
        <tr>
        <td>
        Activation of presynaptic cAMP-dependent protein kinase is required for induction of cerebellar long-term potentiation.
        (<a href="http://www.neuroelectro.org/article/37591/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/10575019/" target="_blank">PubMed</a>)
        </td>
        <td>Linden DJ; Ahn S</td>
        <td>J. Neurosci.</td>
        <td>1999</td>
        <td>1</a></td>
        <td>1</a></td>
        </tr>
    
        <tr>
        <td>
        Selective and regulated gene expression in murine Purkinje cells by in utero electroporation.
        (<a href="http://www.neuroelectro.org/article/72982/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22775058/" target="_blank">PubMed</a>)
        </td>
        <td>Nishiyama J; Hayashi Y; Nomura T; Miura E; Kakegawa W; Yuzaki M</td>
        <td>Eur. J. Neurosci.</td>
        <td>2012</td>
        <td>2</a></td>
        <td>1</a></td>
        </tr>
    
        <tr>
        <td>
        In vivo transduction of murine cerebellar Purkinje cells by HIV-derived lentiviral vectors.
        (<a href="http://www.neuroelectro.org/article/78059/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16516872/" target="_blank">PubMed</a>)
        </td>
        <td>Torashima T; Okoyama S; Nishizaki T; Hirai H</td>
        <td>Brain Res.</td>
        <td>2006</td>
        <td>3</a></td>
        <td>1</a></td>
        </tr>
    
        <tr>
        <td>
        Impaired cerebellar long-term potentiation in type I adenylyl cyclase mutant mice.
        (<a href="http://www.neuroelectro.org/article/79142/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/9655507/" target="_blank">PubMed</a>)
        </td>
        <td>Storm DR; Hansel C; Hacker B; Parent A; Linden DJ</td>
        <td>Neuron</td>
        <td>1998</td>
        <td>4</a></td>
        <td>2</a></td>
        </tr>
    
        <tr>
        <td>
        Chronic ethanol treatment alters AMPA-induced calcium signals in developing Purkinje neurons.
        (<a href="http://www.neuroelectro.org/article/79280/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/10224304/" target="_blank">PubMed</a>)
        </td>
        <td>Netzeband JG; Trotter C; Parsons KL; Gruol DL</td>
        <td>Brain Res.</td>
        <td>1999</td>
        <td>2</a></td>
        <td>1</a></td>
        </tr>
    
        <tr>
        <td>
        Membrane excitability and fear conditioning in cerebellar Purkinje cell.
        (<a href="http://www.neuroelectro.org/article/83523/">NeuroElectro data</a>)
        (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16580140/" target="_blank">PubMed</a>)
        </td>
        <td>Zhu L; Scelfo B; Tempia F; Sacchetti B; Strata P</td>
        <td>Neuroscience</td>
        <td>2006</td>
        <td>4</a></td>
        <td>1</a></td>
        </tr>
    
    </tbody>
    </table>




<br>
<h2>Expert curators for Cerebellum Purkinje cells:</h2>


<p>No assigned expert curators:</p>


        <a class="fancybox submit_button btn btn-medium btn-primary" href="http://www.neuroelectro.org/accounts/login?next=/neuron/18/"><i class="icon-user icon-white"></i> Become a neuron curator (login required)</a>

<br>

<!--
        <div class='row-fluid'>
            <div class='span3 offset1'><a class="btn btn-small btn-primary" href="./curator_ask">Become a curator!</a></div>
        </div>
        <a href="/neuron/18/curate_list/">View current list of papers to curate for this neuron</a>     
-->

<h2>More information about Cerebellum Purkinje cells in other resources:</h2>
<ul>
<li>View information about <i>Cerebellum Purkinje cells</i> across database resources at the <a href="https://neuinfo.org/mynif/search.php?list=cover&amp;q=Cerebellum Purkinje cell" target="_blank">
    Neuroscience Information Framework</a>
</li>


<!--<h2>Genes differentially expressed in Cerebellum Purkinje cells (from the Allen Brain Atlas): </h2>-->
<li>View genes differentially expressed in brain regions containing <i>Cerebellum Purkinje cells</i> from the <a href="http://mouse.brain-map.org/search/show?page_num=0&amp;domain1=8&amp;domain2=512&amp;domain1_threshold=0,50&amp;domain2_threshold=1,50&amp;image_set=P56&amp;search_type=differential" target="_blank">Allen Institute Mouse Brain Atlas</a></li>


 
<li>View defining information for <i>Cerebellum Purkinje cells</i> at
    <a href="http://uri.neuinfo.org/nif/nifstd/sao471801888" target="_blank">
       NeuroLex.org
    </a>
</li>


 
<li>View computational models for <i>Cerebellum Purkinje cells</i> in 
    <a href="http://senselab.med.yale.edu/ModelDb/ModelList.asp?id=271&amp;celldescr=no" target="_blank">
       ModelDB
    </a>
    and ion channels and synaptic currents in
    <a href="http://senselab.med.yale.edu/neurondb/NeuronProp.aspx?id=271" target="_blank">
       NeuronDB
    </a>
</li>


 
<li>View collaboratively developed and standardized computational models containing <i>Cerebellum Purkinje cells</i> at 
    <a href="http://www.opensourcebrain.org/search_custom_field?f[]=33&amp;op[33]=~&amp;v[33][]=sao471801888" target="_blank">
       Open Source Brain
    </a>
</li>



</body>   
</div>
<div id="footer">
<footer>
    
    <table align="center" width="1200px" class="footer_bar">
        <tr>
            <td class="footer_bar">
                <p align="left">
                    &copy; 2012-2015 <a href="https://twitter.com/neuronJoy" target="_blank">Shreejoy Tripathy</a> and 
                    <a href="http://rick.gerk.in/" target="_blank">Richard Gerkin</a>
                    <br> 
                    University of British Columbia and Carnegie Mellon University
                    
                    <!--<br>Funding: NIH, NIF, and the RK Mellon Foundation-->
                    <br>
                    Release Version 3.0
                    <!--Optimized for Mozilla Firefox, Safari, and fast internet connections -->
                </p>
            </td>
            <td class="footer_bar">
                
                <p align="left">Not signed in <a class="btn btn-small btn-primary" href="http://www.neuroelectro.org/accounts/login?next=/neuron/18/"><i class="icon-user icon-white"></i>  Log in</a></p>
                

            </td>
            <td class="footer_bar">
                <p align="right">
                <a href="http://www.neuinfo.org/">
                <img src="https://web.archive.org/web/20151005110219im_/http://neuinfo.org/images/registered-with-nif.png" alt="Registered with NIF" width="125" height="40" style="border:none;">
                </a>
                </p>
            </td>
        </tr>
    </table>
    
</footer>
</div>
</html>
<!--
     FILE ARCHIVED ON 11:02:19 Oct 05, 2015 AND RETRIEVED FROM THE
     INTERNET ARCHIVE ON 15:45:45 Jan 06, 2025.
     JAVASCRIPT APPENDED BY WAYBACK MACHINE, COPYRIGHT INTERNET ARCHIVE.

     ALL OTHER CONTENT MAY ALSO BE PROTECTED BY COPYRIGHT (17 U.S.C.
     SECTION 108(a)(3)).
-->
<!--
playback timings (ms):
  captures_list: 1.141
  exclusion.robots: 0.03
  exclusion.robots.policy: 0.02
  esindex: 0.017
  cdx.remote: 30.525
  LoadShardBlock: 136.773 (3)
  PetaboxLoader3.datanode: 229.792 (4)
  load_resource: 374.208
  PetaboxLoader3.resolve: 86.427
-->